packages/office-addin-lint/src/lint.ts

```diff
@@ -34,12 +34,12 @@ export function performLintCheck(files: string) {
   try {
     const command = getLintCheckCommand(files);
     execCommand(command);
-    usageDataObject.sendUsageDataSuccessEvent("performLintCheck", { exitCode: ESLintExitCode.NoLintErrors });
+    usageDataObject.reportSuccess("performLintCheck()", { exitCode: ESLintExitCode.NoLintErrors });
   } catch (err) {
     if (err.status && err.status == ESLintExitCode.HasLintError) {
-      usageDataObject.sendUsageDataSuccessEvent("performLintCheck", { exitCode: ESLintExitCode.HasLintError });
+      usageDataObject.reportExpectedException("performLintCheck()", "Has lint error", { exitCode: ESLintExitCode.HasLintError });
```

Comment:
Why is this string "Has lint error"? Sorry I didn't look at the code and cannot tell here when looking at the diff.

Reply:
To send an "expectedException", it is required to pass the ```err``` parameter to the functions.
So I passed this string to represent the expected error.

Reply:
Based on the function definitiion:
`public reportExpectedException(method: string, err: Error | string, data: object = {}): void`

I would expect the second parameter to be `err` and not `"Has lint error"`. The exitCode actually distinguishes the cases because exitCode 1 means there is a lint error and 2 for unexpected error.

Let me know if I'm missing something.

Reply:
Sure, a mistake by me. Err should be passed instead of the string